tensorflow_addons/losses/quantiles.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Implements quantiles losses."""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import tensorflow as tf
+
+
+@tf.keras.utils.register_keras_serializable(package='Addons')
+@tf.function
+def pinball_loss(y_true, y_pred, tau=.5):
+    """Computes the pinball loss between `y_true` and `y_pred`.
+
+    `loss = maximum(tau * (y_true - y_pred), (tau - 1) * (y_true - y_pred))`
+
+    In the context of regression this loss yields an estimator of the tau
+    conditional quantile.
+
+    https://en.wikipedia.org/wiki/Quantile_regression
+    https://projecteuclid.org/download/pdfview_1/euclid.bj/1297173840
+
+    Usage:
+
+    ```python
+    loss = pinball_loss([0., 0., 1., 1.], [1., 1., 1., 0.], tau=.1)
+
+    # loss = max(0.1 * (y_true - y_pred), (0.1 - 1) * (y_true - y_pred))
+    #      = (0.9 + 0.9 + 0 + 0.1) / 4
+
+    print('Loss: ', loss.numpy())  # Loss: 0.475
+    ```
+
+    Args:
+
+    y_true: Ground truth values. shape = `[batch_size, d0, .. dN]`
+    y_pred: The predicted values. shape = `[batch_size, d0, .. dN]`
+	tau: Float in [0, 1] or a tensor taking values in [0, 1] and shape =
+      `[d0,..., dn]`.  It defines the slope of the pinball loss. In the context
+      of quantile regression, the value of tau determines the conditional
+      quantile level. When tau = 0.5 this amounts to l1 regression, an
+      estimator of the conditional median (0.5 quantile).
```

Comment:
could we follow the indentation in this file?

https://github.com/tensorflow/addons/blob/master/tensorflow_addons/losses/contrastive.py#L43-L51

Reply:
Also, misses the description of `tau`.